server/src/jobs_tags/jobTagModel.js

```diff
@@ -0,0 +1,20 @@
+import db from '../db/db-config';
+import Sequelize from 'sequelize';
+import Job from '../jobs/jobModel';
+import Tag from '../tags/tagModel';
+
+//generate a JobTag join table 
+let JobTag = db.define('job_tag', {});
+
+Job.belongsToMany(Tag, { through: JobTag });
+Tag.belongsToMany(Job, { through: JobTag });
+
+JobTag.sync()
+  .then(err => {
```

Comment:
again, weird .then structure... perhaps the err should be handled in a catch block instead?